Code/Framework/AzFramework/Tests/ProcessLaunchMain.cpp

```diff
@@ -29,12 +31,54 @@ void OutputArgs(const AzFramework::CommandLine& commandLine)
 
 int main(int argc, char** argv)
 {
+    using namespace ProcessLaunchTestInternal;
+
     const AZ::Debug::Trace tracer;
+    int exitCode = 0;
     {
         AzFramework::CommandLine commandLine;
 
         commandLine.Parse(argc, argv);
-        OutputArgs(commandLine);
+
+        if (commandLine.GetNumSwitchValues("exitCode") > 0)
+        {
+            exitCode = atoi(commandLine.GetSwitchValue("exitCode").c_str());
```

Comment:
[AZStd::from_chars](https://en.cppreference.com/w/cpp/utility/from_chars) can be used here from `#include<AzCore/std/utility/charconv.h>`
That at the very least returns an error code if something goes wrong.
```suggestion
            const AZStd::string& exitCodeStr = commandLine.GetSwitchValue("exitCode");
            AZStd::from_chars(exitCodeStr.begin(), exitCodeStr.end(), exitCode);
```